cli/definitions/cli.ts

```diff
@@ -119,7 +120,19 @@ export interface ILoginCommand extends ICommand {
     accessKey: string;
 }
 
-export interface IPromoteCommand extends ICommand {
+export interface IPackageInfo extends ICommand {
+    description: string;
+    mandatory: boolean;
+    rollout: string;
```

Comment:
Should probably be optional too.

    rollout?: number;

Same feedback applies to IReleaseBaseCommand.rollout.